### DISCUSSION TITLE
Bulk editing of interfaces description

### DISCUSSION THREAD

**Discussion by nikos82** [netbox-community/netbox#10813] [General]
Hello Team,

Recently added a new device and i used the bulk import feature in order to upload the interfaces.
But, by mistake i didn't upload the description for the interfaces.

Trying to reupload the interfaces using the bulk import , adding this time the description column header, i get the below error message : **Interface with this Device and Name already exists.**
-----------------------------------------
device,name,type,description
-----------------------------------------

How can i import the descriptions for the interfaces, avoiding the hard way ?

Thanks for your time.

Regards,
Nikos

**Comment by mtinberg** [comment]
You will either need to delete and re-add the interfaces or write a script to modify the existing ones over the API, or edit them one-by-one by hand.  The CSV import at this time does not support modifying existing records, only creating them for the first time.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: nikos82 ***@***.***>
Sent: Wednesday, November 2, 2022 5:00 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Bulk editing of interfaces descriptions (Discussion #10813)


Hello Team,

Recently added a new device and i used the bulk import feature in order to upload the interfaces.
But, by mistake i didn't upload the description for the interfaces.

Trying to reupload the interfaces using the bulk import , adding this time the column header, i get the below error message : Interface with this Device and Name already exists.
device,name,type,description

How can i import the descriptions for the interfaces, avoiding the hard way ?

Thanks for your time.

Regards,
Nikos

**Reply by jeremystretch** [reply]
> The CSV import at this time does not support modifying existing records, only creating them for the first time.

FYI this ability is coming in NetBox v3.4; see #7961.

**Comment by nikos82** [comment]
Hi Team,

Thanks' a lot for the clarifications.

Have a nice day.
